lib/anyplayer/players/rdio_mac.rb

```diff
@@ -47,22 +47,21 @@ def album
 
   def playing?
     playing = rdio 'return player state is playing'
-    playing == "true"
+    playing == 'true'
   end
 
   def launched?
-    %x(osascript -e 'app "rdio" is running').rstrip == "true"
+    %x(osascript -e 'app "rdio" is running').rstrip == 'true'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.<br>Do not use `%x` unless the command string contains backquotes.